src/appleseed.common/application/application.cpp

```diff
@@ -131,6 +131,9 @@ void Application::check_compatibility_with_host(Logger& logger)
 
 namespace
 {
+    static char g_root_path[FOUNDATION_MAX_PATH_LENGTH + 1] = { 0 };
+    static bool g_is_root_path_set = false;
+
     // Compute the root path of the application.  Return true if the root path could be
```

Comment:
Dupliacted blank space before `Return `. But that's out of the scope of your PR :)

Reply:
That was for alignment :) It's a [bad habit](https://www.google.com/search?q=two+spaces+between+sentences), I'll remove the redundant space.